### DISCUSSION TITLE
Can I implement Skip in an already started Swift project

### DISCUSSION THREAD

**Discussion by This-Marinov-guy** [skiptools/skip#149] [Q&A] [answered]
Hello, just a simple one - can I add skip to a small project I have? It would be a nice feature. Great product by the way!

**Comment by aabewhite** [comment, marked as answer]
Hi - Yes you can, though it's typically not easy. The docs explain: 

https://skip.tools/docs/gettingstarted/#existing_development

Please don't hesitate to reach out to us for support along the way!

**Reply by This-Marinov-guy** [reply]
Thanks for the fast response!
